src/App.js

```diff
@@ -520,18 +524,21 @@ class App extends React.Component<Props, State> {
   onOpenWheelchairAccessibility = () => {
     if (this.props.featureId) {
       this.setState({ modalNodeState: 'edit-wheelchair-accessibility' });
+      trackModalView('edit-wheelchair-accessibility');
     }
   };
 
   onOpenToiletAccessibility = () => {
     if (this.props.featureId) {
       this.setState({ modalNodeState: 'edit-toilet-accessibility' });
+      trackModalView('edit-wheelchair-accessibility');
```

Comment:
wouldn't this be `edit-toilet-accessibility`?

Reply:
Fixed, thanks!